addons/hide-stage/style.css

```diff
@@ -16,10 +16,13 @@
   border-radius: 0;
 }
 
-.sa-stage-hidden [class*="gui_stage-and-target-wrapper_"],
+/* [class*="gui_flex-wrapper_"] is for specificity over hide-flyout */
+.sa-stage-hidden [class*="gui_flex-wrapper_"] [class*="gui_stage-and-target-wrapper_"],
 .sa-stage-hidden [class*="stage-wrapper_stage-wrapper_"]:not([class*="stage-wrapper_full-screen_"]),
 .sa-stage-hidden [class*="gui_target-wrapper_"] {
   padding: 0;
+  /* override hide-flyout */
+  padding-inline: 0;
```

Comment:
This line doesn't appear to have any effect - the specificity change is enough to fix the bug.

Reply:
Removed, thanks